routes/favicon.coffee

```diff
@@ -0,0 +1,27 @@
+request = require 'request'
+fs = require 'fs'
+tmpdir = require 'os'
+	.tmpdir()
```

Comment:
You've just assigned a string to `tmpdir`. You're probably aware of that but I don't think an assignment like that should be with the module imports. I think instead you should do `os = require 'os`, and then invoke `os.tmpdir()` later below.